cpp/ps0004/p0004.cpp

```diff
@@ -1,33 +1,33 @@
 #include <iostream>
 using namespace std;
 int main() {
-    int number;
-   cout << "Enter a number between 32 and 126: ";
-      cin >> number;
+    // prompting user for number 1/1.5
+    float number;
+    cout << "Enter a number between 32 and 126: ";
+    cin >> number;

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input